os/board/Kconfig

```diff
@@ -191,6 +192,12 @@ config ARCH_IRQBUTTONS
 	---help---
 		"Support interrupts on button presses and releases."
 
+config ARCH_HAVE_RAM_KERNEL_TEXT
+	bool "Kernel text in RAM region"
```

Comment:
As I recommend, this config should not be controlled by user. And naming `ARCH_HAVE_XXX` means architecture controls the config. Please remove `"Kernel text in RAM region"`.